specs/agents/mobile/configuration.md

```diff
@@ -0,0 +1,15 @@
+## Mobile Configuration
+
+This document describes the [central configuration](../configuration.md) parameters used in mobile agents. These values
+can be set through Kibana's APM Settings.
+
+### `enable_automatic_instrumentation` configuration
```

Comment:
This looks very similar to the [`instrument`](https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html#config-instrument) config option. Can we use that name instead?